packages/admin-panel/src/VizBuilderApp/api/queries/useSearchDataSources.js

```diff
@@ -26,3 +25,9 @@ export const useSearchDataSources = ({ search, type = 'dataElement', maxResults
       keepPreviousData: true,
     },
   );
+  const mappedData = result?.data?.map(value => ({ ...value, type }));
```

Comment:
The actual usage of this data expects type to be included, but since we can't get that directly from the api anymore it needs to be added

We could possibly refactor the usage in `DataElementDataLibrary.js` so we don't need to do this, but for now I've just made the return from this function match the prior implementation